### DISCUSSION TITLE
AM was added to Topgrade

### DISCUSSION THREAD

**Discussion by sbstn87** [ivan-hc/AM#12] [General]
Hi,

AM was just added to [topgrade](https://github.com/topgrade-rs/topgrade):
https://github.com/topgrade-rs/topgrade/pull/328

"Keeping your system up to date usually involves invoking multiple package managers. This results in big, non-portable shell one-liners saved in your shell. To remedy this, Topgrade detects which tools you use and runs the appropriate commands to update them."

**Comment by ivan-hc** [comment]
https://github.com/topgrade-rs/topgrade/pull/328#issuecomment-1493498789
